de.fu_berlin.inf.dpp.server/src/de/fu_berlin/inf/dpp/server/ServerContextFactory.java

```diff
@@ -89,10 +90,10 @@ private void addOptionialCoreInterfaceImplementations(MutablePicoContainer c) {
     }
 
     private void addAdditionalComponents(MutablePicoContainer c) {
-        // c.addComponent(JoinSessionRequestHandler.class);
         // c.addComponent(ServerFeatureAdvertiser.class);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=290&issues=AWbKHVp7LpqqSUlcAilm&open=AWbKHVp7LpqqSUlcAilm)

[ISSUE_KEY]: AWbKHVp7LpqqSUlcAilm